deployment/environment/crib/ccip_deployer.go

```diff
@@ -0,0 +1,121 @@
+package crib
+
+import (
+	"context"
+	"errors"
+
+	chainsel "github.com/smartcontractkit/chain-selectors"
+	jobv1 "github.com/smartcontractkit/chainlink-protos/job-distributor/v1/job"
+
+	"github.com/smartcontractkit/chainlink/deployment/environment/devenv"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	ccipdeployment "github.com/smartcontractkit/chainlink/deployment/ccip"
+	"github.com/smartcontractkit/chainlink/deployment/ccip/changeset"
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+	"github.com/smartcontractkit/chainlink/v2/core/services/relay"
+)
+
+// DeployHomeChainContracts deploys the home chain contracts so that the chainlink nodes can be started with the CR address in Capabilities.ExternalRegistry
+func DeployHomeChainContracts(lggr logger.Logger, envConfig devenv.EnvironmentConfig, homeChainSel uint64) (deployment.CapabilityRegistryConfig, deployment.AddressBook, error) {
+	chains, err := devenv.NewChains(lggr, envConfig.Chains)
+	if err != nil {
+		return deployment.CapabilityRegistryConfig{}, nil, err
+	}
+
+	ab := deployment.NewMemoryAddressBook()
+	capReg, err := ccipdeployment.DeployCapReg(lggr, ab, chains[homeChainSel])
+	if err != nil {
+		return deployment.CapabilityRegistryConfig{}, nil, err
+	}
+	if err != nil {
+		return deployment.CapabilityRegistryConfig{}, nil, err
+	}
+	evmChainID, err := chainsel.ChainIdFromSelector(homeChainSel)
+	if err != nil {
+		return deployment.CapabilityRegistryConfig{}, nil, err
+	}
+	return deployment.CapabilityRegistryConfig{
+		NetworkType: relay.NetworkEVM,
+		EVMChainID:  evmChainID,
+		Contract:    capReg.Address,
+	}, ab, nil
+}
+
+func DeployCCIPAndAddLanes(lggr logger.Logger, envCfg devenv.EnvironmentConfig, homeChainSel, feedChainSel uint64, ab deployment.AddressBook) error {
```

Comment:
@AnieeG @0xAustinWang This function needs to return whatever data is needed later on for running the tests.

The return struct should definitely contain:
* addressBook 

I assume we can use the LoadChainState function to initialize EnvState struct from the addressBook.

Ideally we should use structs can be be serialized into json.
AddressBook is fine, but there are other objects like `deployment.Environment{}` in @0xAustinWang [spike here](https://github.com/smartcontractkit/chainlink/blob/e9d7280445bcb4ef14541035e77416ceb54449fc/integration-tests/load/ccip/ccip_load.go#L51C12-L51C36) that contains business logic and client initialization. So we should isolate business logic and just pass data between CRIB and your tests.